packages/itwinui-react/src/core/Alert/Alert.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
this should be done per-line instead of for the whole file

Reply:
Changed